packages/elements/src/components/table/table.stories.mdx

```diff
@@ -148,6 +185,23 @@ If set, this prop will ensure that in narrow view mode the cell takes up both co
 
 <RenderHtmlMarkup component="TableCell" story="TableCell - Narrow mode: full width" />
 
+### Narrow order 
+
+If you require to change the order of your cells in the mobile/narrow view, you can use the `narrowOrder` property to change the cell's order in the column.
+
+The order can be specified between 1 and 12.
+
+<Canvas>
+  <Story name="TableCell - Narrow order">
+    <TableCell narrowOrder={3}>
```

Comment:
How do I do narrowOrder if I am non-React? Just a line of commentry - add this class etc